sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_client.py

```diff
@@ -517,8 +517,8 @@ def list_dimension_values(
         .. admonition:: Example:
 
             .. literalinclude:: ../samples/sample_queries.py
-                :start-after: [START list_dimension_values]
-                :end-before: [END list_dimension_values]
+                :start-after: [START list_anomaly_dimension_values]
```

Comment:
I think samples will need renaming update as well

Reply:
Hah. You are right. I forgot to add samples changes into the PR. Added.